nncf/sparsity/schedulers.py

```diff
@@ -84,8 +84,7 @@ def __init__(self, sparsity_algo, params=None):
                                "but steps_per_epoch was not set in config. Will only start updating "
                                "sparsity level after measuring the actual steps per epoch as signaled "
                                "by a .epoch_step() call.")
-
-        self._set_sparsity_level()
+        #self._set_sparsity_level()
```

Comment:
Is that will be removed or it means that it could be optionally called (why if so)?

Reply:
It will be removed.